src/protocol/cmds.rs

```diff
@@ -8,7 +8,21 @@ use pyo3::{
 
 use libparsec::protocol::{authenticated_cmds, invited_cmds};
 
-use crate::protocol::*;
+use crate::protocol::{
```

Comment:
This is typically the kind of thing we should avoid: we want to re-expose this stuff in a simple way and not having to get error from this place next time we add a new command

Reply:
Can you relace this by:
```rust
use crate::protocol;
[...]
  AnyCmdReq::InviteList(x) => protocol::InviteListReq(x).into_py(py),
```